lib/reactions/on_wallet_sync_status_change.dart

```diff
@@ -1,21 +1,31 @@
+import 'package:cake_wallet/di.dart';
+import 'package:cake_wallet/entities/wake_lock.dart';
 import 'package:mobx/mobx.dart';
 import 'package:cw_core/transaction_history.dart';
 import 'package:cw_core/wallet_base.dart';
 import 'package:cw_core/balance.dart';
 import 'package:cw_core/transaction_info.dart';
 import 'package:cw_core/sync_status.dart';
+import 'package:flutter/services.dart';
 
 ReactionDisposer _onWalletSyncStatusChangeReaction;
 
 void startWalletSyncStatusChangeReaction(
     WalletBase<Balance, TransactionHistoryBase<TransactionInfo>,
             TransactionInfo>
         wallet) {
+  final WakeLock _wakeLock = getIt.get<WakeLock>();
```

Comment:
Don't define type of `final` variable, because it's necessary as dart will get type of your value, as that's `final` variable, then you will not change the value